assets/src/edit-story/components/canvas/displayLayer.js

```diff
@@ -31,7 +31,8 @@ const DisplayPageArea = styled(PageArea).attrs({
   className: 'container',
   overflowAllowed: false,
 })`
-  background-color: ${({ theme }) => theme.colors.fg.v1};
+  background-color: ${({ theme, backgroundColor }) =>
+    backgroundColor || theme.colors.fg.v1};
```

Comment:
I don't know if this needs to be added to the `DisplayLayer` too. @dvoytenko?

Reply:
Perhaps only in the `DisplayLayer`? Adding this twice would cause issues once opacity comes to play.

If I understand correctly then the percentage next to the color means opacity.

Reply:
Surprisingly, adding it only to the `DisplayLayer` did nothing on screen, as the `FrameLayer` has a opaque background. Maybe I just don't understand the naming here.

Reply:
Hmm, maybe the frames layer shouldn't have the background color set then? @dvoytenko Any reason for the `framesLayer` to have the background color set?

Reply:
Setting background on the `DisplayPageArea` looks exactly right to me. It shouldn't be set anywhere else, I don't believe. `FrameLayer` is only singularly responsible for drawing frames (bounds) of elements and floating stuff such as icons - but nothing else.

Reply:
But if there's frame opaque background - I believe that's just a bug. I'll take a look.

Reply:
Yep, seeing background set [here](https://github.com/google/web-stories-wp/blob/58045a006504ee551fb5c51ee3624e9e9db4a13e/assets/src/edit-story/components/canvas/framesLayer.js#L41). Definitely a copy/paste bug. Surprised we didn't notice it earlier.

Reply:
Also odd that we're using a `fg` color here.

Reply:
Okay, I'll remove bg from the frames layer.

Reply:
Done in ef3a047d.